client/src/components/friend-list.js

```diff
@@ -0,0 +1,98 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { css } from 'styled-components/macro';
+import { Empty, Spin, Button, Row, Col } from 'antd';
+import { PostStore, AuthStore, FriendsStore } from '../stores';
+import { inject } from '../utils';
+import FriendCard from './friend-card';
+
+// Comment out username for now as API does not supply this field
+const CardMetaTitle = ({ displayName, extra }) => (
+  <Fragment>
+    {displayName}{' '}
+    <span
+      css={css`
+        float: right;
+      `}
+    >
+      {extra}
+    </span>
+    <div
+      css={css`
+        padding-top: 16px;
+      `}
+    >
+      Hello, my name is {displayName}
```

Comment:
The font-size and weight are bad.

Follow the guideline here
- https://ant.design/docs/spec/font-cn

Additional resources:
- https://airbnb.design/building-a-visual-language/

Reply:
Font-size and weight are reformatted.

Reply:
Replace placeholder with actual author bio